qa/workunits/rados/test_pool_quota.sh

```diff
@@ -31,8 +31,28 @@ sleep 30
 
 rados -p $p put three /etc/passwd
 
+
+#one pool being full does not block a different pool
+
+pp=`uuidgen`
+
+ceph osd pool create $pp 12
+
+# set objects quota 
+ceph osd pool set-quota $pp max_objects 10
+sleep 30
+
+for f in `seq 1 10` ; do
+ rados -p $pp put obj$f /etc/passwd
+done
+
+sleep 30
+
+rados -p $pp put onemore /etc/passwd  &
+
 # done
 ceph osd pool delete $p $p --yes-i-really-really-mean-it
```

Comment:
you'll want to put the waits before the pool deletes so odd scheduling doesn't cause spurious failures